README.md

```diff
@@ -0,0 +1,154 @@
+# uwv\_dynamic\_model: Implementation of a dynamic model for undewater vehicles
+
+Assumptions:
+
+ - A vehicle, considered a rigid body, is represented by a point in space, been this point the center of gravity (CoG).
+ - The CoG is the origin of the right-hand side body-frame.
+ - There is no representation of multiples objects, neither contact or interaction betwen them.
+ - There is no representation of the environment (floor, surface, water currents, waves, etc)[^TODO].
+ - The vehicle is considered to be surrended by fluid infinitly in all direction.
+ - The gravity field is considered uniform.
+
+## General presentation
+
+The present dynamic model implements the formulation proposed mainly by [Fossen][].
+
+It implements the dynamic and kinematic of a vehicle and its hydrodynamic effect. See [UWV Parameters](#parameters) section.
+There are three induced hydrodynamic efforts:
+
+ 1. Added mass due to the inertia of the surrouding fluid.
+ 2. Damping effect.
+ 3. Restoring gravitational forces (weight and buoyancy).
+
+The hydrodynamic effect is embedded in the parameters passed to the model.
+It is required that the parameters are identified previously. (The code doesn't estimated them).
+
+In order to add some flexibility on how each term would affect the model, the present library has 
+a ModelType property. It allows the user to choose a specific formulation of the damping term or 
+to ignore the Corilois effect for a simplified model, for example. See [UWV Parameters](#parameters) section.
+
+The control efforts applied in the vechile are considered to be in the CoG.
+In case a set of propellers need to be represented, the dynamic model just requires
+the resultant effect (in terms of forces and torques) in the CoG.
+
+In the 6 degree of freedom (DOF) formulation, the order of representing each DOF is:
+
+ 1. linear x
+ 2. linear y
+ 3. linear z
+ 4. angular x
+ 5. angular y
+ 6. angular z
+
+The efforts applied as well as the velocities (both linear as angular) are represented in body-frame.
```

Comment:
I think if it is in the library and you don't use the RBS type it is absolutely fine.